### ISSUE TITLE
v1.11.0 : constant 10% cpu usage from kasm_manager.so

### ISSUE THREAD

**Issue by Daryes** [kasmtech/workspaces-issues#247]
Hello,

As said in the title, when I upgraded from Kasm Workspaces  v1.9.0 to v1.11.0, I discovered later that the kasm_manager.so process  had a constant 8% to 14% cpu usage
It happens immediately after start, without stopping (even days after). 
The logs file gives no information, they are devoid of anything aside the usual _healthcheck flood.
The cpu usage didn't occur when using v1.9.0

This aside, Kasm works without any error, desktop can be started correctly.

some informations:
* Docker version 20.10.17, build 100c701
* Docker Compose version v2.6.1
* kernel 4.15.0-189-generic
* Ubuntu 18.04.6 LTS (Bionic Beaver)
* installation is on a single VM, 4vCPU, 4Gb ram
* host cpu:  [Xeon E-2144G](https://ark.intel.com/content/www/us/en/ark/products/134862/intel-xeon-e2144g-processor-8m-cache-up-to-4-50-ghz.html)

To reproduce : 
```
# start kasm
sudo /opt/kasm/bin/start

# use top or htop to check the cpu usage
htop
```

**Comment by neiaberau** [comment]
same, and    strace tells me it constantly does this, whatever this does ...

tat("/tmp/_MEIV4kPkS/sqlalchemy/dialects/mssql/pymssql.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy/dialects/mssql/zxjdbc.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy/dialects/mssql/__init__.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy_utils/types/uuid.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy_utils/types/bit.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy_utils/types/weekdays.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy_utils/types/__init__.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy_utils/view.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/sqlalchemy_utils/__init__.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/data/model.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/pythonjsonlogger/__init__.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/pythonjsonlogger/jsonlogger.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/log/handlers.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/provider.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/providers/base.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/azure/identity/_auth_record.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/azure/core/_version.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
stat("/tmp/_MEIV4kPkS/azure/core/configuration.py", 0x7ffee01b0130) = -1 ENOENT (No such file or directory)
etc.

**Comment by j-travis** [comment]
Thanks for reporting. We have been looking into this and will report when we have an update

**Comment by Yetangitu** [comment]
Any news on this front? The root cause seems to be the fact that the whole thing is written in Python after which _PyInstaller_ was used in `--one-file` mode to create a single-file "executable" from which the _.py_ files have been stripped, probably in an attempt to obfuscate the source. This does not really work since the _.pyc_ files have been included and can be decompiled. Meanwhile something seems to be polling for those missing _.py_ files *every second* as can be gleaned from tracing the process:

```
# timeout 30s strace -t -f -p 1155 -o /tmp/dump;awk '/ENOENT/ { print $2 }' /tmp/dump |uniq
strace: Process 1155 attached with 4 threads
strace: Process 1155 detached
strace: Process 1246 detached
strace: Process 1247 detached
strace: Process 1261 detached
22:43:01
22:43:02
22:43:03
22:43:04
22:43:05
22:43:06
22:43:07
22:43:08
22:43:09
22:43:10
22:43:11
22:43:12
22:43:13
22:43:14
22:43:15
22:43:16
22:43:17
22:43:18
22:43:19
22:43:20
22:43:21
22:43:22
22:43:23
22:43:24
22:43:25
22:43:26
22:43:27
22:43:28
22:43:29
22:43:30
22:43:31
```
No wonder that the thing is slow when it is _stat(3)_-ing _6552 files_ every second.

Do mind that the thing is calling `epoll_wait` at least twice per second, one of these runs `stat(3)` over what seems to be the TOC for the PyInstaller zipfile:

```
# awk '/epoll_wait/ { print $2, $3}' /tmp/dump 
22:43:01 epoll_wait(9,
22:43:02 epoll_wait(9,
22:43:03 epoll_wait(9,
22:43:03 <...
22:43:04 epoll_wait(9,
22:43:05 epoll_wait(9,
22:43:06 epoll_wait(9,
22:43:07 epoll_wait(9,
22:43:08 epoll_wait(9,
22:43:08 <...
22:43:09 epoll_wait(9,
22:43:10 epoll_wait(9,
22:43:11 epoll_wait(9,
22:43:12 epoll_wait(9,
22:43:13 epoll_wait(9,
22:43:13 <...
22:43:14 epoll_wait(9,
22:43:15 epoll_wait(9,
22:43:15 epoll_wait(9,
22:43:15 epoll_wait(9,
22:43:15 epoll_wait(9,
22:43:15 <...
22:43:16 epoll_wait(9,
22:43:16 epoll_wait(9,
22:43:17 epoll_wait(9,
22:43:18 epoll_wait(9,
22:43:18 <...
22:43:19 epoll_wait(9,
22:43:19 epoll_wait(9,
22:43:19 epoll_wait(9,
22:43:19 epoll_wait(9,
22:43:19 epoll_wait(9,
22:43:20 epoll_wait(9,
22:43:21 epoll_wait(9,
22:43:22 epoll_wait(9,
22:43:23 epoll_wait(9,
22:43:23 <...
22:43:24 epoll_wait(9,
22:43:25 epoll_wait(9,
22:43:26 epoll_wait(9,
22:43:27 epoll_wait(9,
22:43:27 epoll_wait(9,
22:43:27 epoll_wait(9,
22:43:28 epoll_wait(9,
22:43:28 <...
22:43:29 epoll_wait(9,
22:43:30 epoll_wait(9,
22:43:31 epoll_wait(9,
22:43:31 epoll_wait(9,
22:43:31 epoll_wait(9,
22:43:31 epoll_wait(9,
22:43:31 <...
22:43:31 epoll_wait(9,
```

Is something watching the PyInstaller zipfile (`PYZ-00.pyz`) for changes? It looks like something is reading the TOC and looking for the source files. Did you ship a debug build set for live updates? I tried changing the DEBUG log settings in the config tile to INFO but this did not make any difference wrt. the above.

Due to the absence of real source files it is hard for outsiders to find the cause of this rather annoying problem.

**Comment by di-org** [comment]
I don't have anything technical to add - just wanted to +1 that I have the same issue, and see the same results with strace.

**Comment by j-travis** [comment]
Thank you all for the detailed feedback. With your help, we've been able to identify and correct the issue.
Its available in the developer preview builds now if you want to verify: https://kasmweb.com/docs/latest/developers/builds.html

You should notice that your system should be closer to 0% CPU utilization now, with slight upticks to ~1-2% at regular intervals  when the system is doing actual work

**Comment by Yetangitu** [comment]
Installed the preview, this problem seems to be solved.

Just for the record, what was the cause?

**Comment by j-travis** [comment]
autoreload was enabled in the web servers :facepalm . The feature constantly scans the disk look for changes to the code to trigger a reload.  Its helpful tool for development but should not have been left on for the release builds.

**Comment by Yetangitu** [comment]
The problem is not completely solved though:

kasm_share.so:

```
# docker ps|grep kasm_share
6d131df645d4   kasmweb/share:develop       "/bin/sh -c '/usr/bi…"   47 minutes ago   Up 46 minutes (healthy)   8182/tcp                                        kasm_share

root@beta:/tmp# ps aux|grep kasm_share.so|egrep -v 'startup|grep'|awk '{print $2}'
7596
root@beta:/tmp# timeout 1 strace -f -p 7596
strace: Process 7596 attached with 3 threads
[pid  7596] epoll_wait(9,  <unfinished ...>
[pid  7702] futex(0x7fd68c00fa90, FUTEX_WAIT_BITSET_PRIVATE|FUTEX_CLOCK_REALTIME, 0, NULL, FUTEX_BITSET_MATCH_ANY <unfinished ...>
[pid  7701] select(9, [8], [], [], {tv_sec=0, tv_usec=686395} <unfinished ...>
[pid  7596] <... epoll_wait resumed>[], 3, 410) = 0
[pid  7596] stat("/usr/bin/importlib/_bootstrap.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/importlib/_bootstrap_external.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/codecs.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/encodings/aliases.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/encodings/__init__.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/encodings/utf_8.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/share_server.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/encodings/latin_1.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/abc.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/io.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("struct.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("PyInstaller/loader/pyimod01_archive.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("PyInstaller/loader/pyimod02_importers.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/stat.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/_collections_abc.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/genericpath.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/posixpath.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/posixpath.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/os.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/types.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/enum.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/sre_constants.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/sre_parse.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/sre_compile.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/operator.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/keyword.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/heapq.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/reprlib.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/collections/__init__.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/functools.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/copyreg.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/re.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/fnmatch.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/ntpath.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/urllib/__init__.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/warnings.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/urllib/parse.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/pathlib.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/token.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/base_library.zip/tokenize.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("PyInstaller/loader/pyimod03_ctypes.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/lib-dynload/_ctypes.cpython-38-x86_64-linux-gnu.so", {st_mode=S_IFREG|0755, st_size=140592, ...}) = 0
[pid  7596] stat("/usr/bin/ctypes/_endian.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/ctypes/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/lib-dynload/_opcode.cpython-38-x86_64-linux-gnu.so", {st_mode=S_IFREG|0755, st_size=14672, ...}) = 0
[pid  7596] stat("/usr/bin/opcode.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/dis.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/collections/abc.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/importlib/_bootstrap.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/importlib/_bootstrap_external.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/importlib/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/importlib/machinery.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/linecache.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/inspect.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/importlib/abc.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/contextlib.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/importlib/util.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/_compression.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/_weakrefset.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/threading.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/lib-dynload/_bz2.cpython-38-x86_64-linux-gnu.so", {st_mode=S_IFREG|0755, st_size=28520, ...}) = 0
[pid  7596] stat("/usr/bin/bz2.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/lib-dynload/_lzma.cpython-38-x86_64-linux-gnu.so", {st_mode=S_IFREG|0755, st_size=45832, ...}) = 0
[pid  7596] stat("/usr/bin/lzma.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/shutil.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/zipfile.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/weakref.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/pkgutil.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/platform.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/datetime.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/xml/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/xml/parsers/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/xml/parsers/expat.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/plistlib.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/errors.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/string.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/quoprimime.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base64.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/base64mime.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/quopri.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/encoders.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/charset.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/header.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/bisect.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/random.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/selectors.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/socket.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/base_library.zip/locale.py", 0x7fff86e7c4e0) = -1 ENOTDIR (Not a directory)
[pid  7596] stat("/usr/bin/calendar.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/_parseaddr.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/utils.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/_policybase.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/feedparser.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/email/parser.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/tempfile.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/textwrap.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/extern/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/jaraco/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/jaraco/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/typing.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/importlib/resources.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/zipp.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/importlib_resources/_compat.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/importlib_resources/abc.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/importlib_resources/_common.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/importlib_resources/_legacy.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/importlib_resources/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such file or directory)
[pid  7596] stat("/usr/bin/pkg_resources/_vendor/importlib_resources/__init__.py", 0x7fff86e7c4e0) = -1 ENOENT (No such 
...
...
... (you get my drift)
```

**Comment by j-travis** [comment]
Thank you, this should have been corrected as well in the latest build

**Comment by Daryes** [comment]
Sorry, took some time to come back
It is fixed at least on v1.13.1
